features/admins/users/admin_abuse_users.feature

```diff
@@ -183,6 +183,24 @@ Feature: Admin Abuse actions
       And there should be no bookmarks on the work "Not Spam"
       And there should be no comments on the work "Not Spam"
 
+  Scenario: A permabanned spammer's comments' replies from others should stay visible
+    Given I have a work "Generic Work"
+      And a comment "I like spam" by "Spamster" on the work "Generic Work"
+      And a reply "I don't :(" by "NotSpamster" on the work "Generic Work"
```

Comment:
Could you make a second comment thread with a comment from Spamster and a reply from Spamster to test the Error 500 from the Jira issue? It occurs when deleting the creations, so it seems convenient to cover it in the same scenario.

Reply:
Note: To make this work with the "I view the latest comment" step, the spammers comment thread would have to be the _first_ comment thread on the work.